addons/trenches/stringtable.xml

```diff
@@ -222,5 +226,220 @@
+            <English>The fatigue factor when building trenches (0 to deactive fatigue use)</English>
+            <German>Der Ausdauerkoeffizient beim Bauen von Gräben (0 zum Deaktivieren)</German>
+            <Russian>Как сильно снижается стамина от копания (0 чтобы не снижалась)</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_ShortEnvelopeDigTime">
+            <English>Dig time (in seconds) for short envelope trenches</English>
+            <German>Zeit (in Sekunden) um kurze runde Gräben zu graben</German>
+            <Russian>Время (в секундах) копания короткого окопа</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_SmallEnvelopeDigTime">
+            <English>Dig time (in seconds) for small envelope trenches</English>
+            <German>Zeit (in Sekunden) um kleine runde Gräben zu graben</German>
+            <Russian>Время (в секундах) копания малого окопа</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_BigEnvelopeDigTime">
+            <English>Dig time (in seconds) for big envelope trenches</English>
+            <German>Zeit (in Sekunden) um große runde Gräben zu graben</German>
+            <Russian>Время (в секундах) копания большого окопа</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_GiantEnvelopeDigTime">
+            <English>Dig time (in seconds) for giant envelope trenches</English>
+            <German>Zeit (in Sekunden) um riesige runde Gräben zu graben</German>
+            <Russian>Время (в секундах) копания гигантского окопа</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_VehicleEnvelopeDigTime">
+            <English>Dig time (in seconds) for vehicle envelope trenches</English>
+            <German>Zeit (in Sekunden) um runde Fahrzeuggräben zu graben</German>
+            <Russian>Время (в секундах) копания окопа для техники</Russian>
+        </Key>
+        <Key ID="STR_ACE_Trenches_HelpDigging">
+            <English>Help digging</English>
+            <German>Mit graben</German>
+        </Key>
+        <Key ID="STR_ACE_Trenches_stopBuildingAtFatigueMax_displayName">
+            <English>Enable Stop on Fatigue</English>
+            <German>Aktiviere Ausdauerbedingter grabe stop</German>
+        </Key>
+        <Key ID="STR_ACE_Trenches_stopBuildingAtFatigueMax_tooltip">
+            <English>If enabled the digger will stop digging if his fatigue reaches 0.</English>
+            <German>Wenn aktiviert hören die Spieler auf zu graben wenn ihre Ausdauer 0 erreicht.</German>
+        </Key>
+        <Key ID="STR_ACE_Trenches_enableAutomaticFilePath_displayName">
+            <English>Enable automatic file path for textures</English>
+            <German>Aktiviert automatische textur ausfüllung</German>
+        </Key>
+        <Key ID="STR_ACE_Trenches_enableAutomaticFilePath_tooltip">
+            <English>If enabled the mod will try to fill in an automatic path and prefix for the texture files. (WARNING!!!) This can cause errors!</English>
```

Comment:
```suggestion
            <English>If enabled the mod will try to fill in an automatic path and prefix for the texture files (WARNING: Can cause misconfiguration!).</English>
```
Unless it can actually cause errors, in which case, why even have it?

Reply:
Yes it can create errors. It is there so that terrains that are not in CfgWorlds can also use the surface texture, there is a reson why there is option to turn it on/off!

Reply:
How can it cause errors? Why doesn't it fall-back to non-error state even if texture is wrong? I am speaking about script errors (RPT), not visual.